yt/visualization/volume_rendering/transfer_functions.py

```diff
@@ -224,13 +224,13 @@ def show(self):
         >>> tf.add_gaussian(-9.0, 0.01, 1.0)
         >>> tf.show()
         """
-        import pylab
+        import matplotlib.pyplot as plt
```

Comment:
This one doesn't have
```
matplotlib.use("Agg")
```
Is it a mistake?

Reply:
🤷🏻‍♂️